hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -38,4 +38,19 @@ objects:
     channel: ${CHANNEL}
     name: splunk-forwarder-operator
     source: splunk-forwarder-operator-catalog
-    sourceNamespace: splunk-forwarder-operator
\ No newline at end of file
+    sourceNamespace: splunk-forwarder-operator
+
+- apiVersion: splunkforwarder.managed.openshift.io/v1alpha1
+  kind: SplunkForwarder
+  metadata:
+    name: example-splunkforwarder
```

Comment:
example? Should we change the name here?

Reply:
this is what i get for just copying the example instead of rewriting it :) fixed